src/js/react_views/IntlHelperBarView.jsx

```diff
@@ -88,6 +88,12 @@ var IntlHelperBarView = React.createClass({
         this.fireCommand('locale ru_RU; levels');
       }.bind(this)
     }, {
+      text: 'Українська',
```

Comment:
ah good! you found this -- this is one of the two spots to update if you are adding a new locale